apps/challenges/urls.py

```diff
@@ -21,5 +21,7 @@
         name='get_all_challenges'),
     url(r'challenge/(?P<pk>[0-9]+)/',
         views.get_challenge_by_pk, name='get_challenge_by_pk'),
+    url(r'challenge$', views.get_challenges_based_on_teams,
```

Comment:
I have kept challenge intentionally, so that till this commit all apis are like

```
* challenges/challenge/:challenge_id
* challenges/challenge/present
* challenges/challenge?participant_team=1
```